### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by HSMathebula**
You can find the features added for this pull request below.
The project was built by @huseyinbiyik @HSMathebula @ahzamir

- Create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. decode_char(".-") returns "A").

- Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").

- Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g.

`  decode("-- -.--   -. .- -- .")`
returns `"MY NAME"`).

- Reuse methods whenever possible. Make sure that your code is DRY.

- Once your code is ready use it to decode the message from the old bottle:

`      .-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...`

### CODE REVIEW

File: decoder.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  word.split.each do |morse|
+    decoded_char = decode_morse_character(morse)
+    decoded_word += decoded_char.to_s
+  end
+  decoded_word
+end
+
+# method to decode the entire message in Morse code, takes a string parameter, and return the string
+# representation. Every word will be separated by 3 spaces.
+def decode(message)
```
Comment (rloterh) [inline decoder.rb:64]: - [ ] Your program is not producing the right decode  output in the console. Only a single output is printed when your app is run. Kindly compare the attached screenshots and ensure that the full decoded messages is printed in the console.

**Your output:**
image


**Expected output:**
image

File: README.md [hunk 2]
```diff
@@ -1,56 +1,43 @@
...
+2- Get the directory that you want to clone the repository. <br>
+3- Open the command prompt in this directory. <br>
+4- Write `git clone https://github.com/HSMathebula/morse-decoder.git` <br>
+5- Go to the repository folder in your command prompt `cd morse-decoder`. <br>
 
-### Run tests
+### Prerequisites
 
-### Deployment
+IRB or VSCode
```
Comment (rloterh) [inline README.md:28]: - [x] OPTIONAL
Kindly add instruction in the ReadMe on how to run your app. This will make your ReadMe appear more professional.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi  Zamir, Happen & Huseyin, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Added required classes

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

You can also consider:
N/A

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by huseyinbiyik** [comment]
Thank you for the review. We fixed it 👨‍💻👍

File: README.md (hunk 2 above)
Comment (rloterh) [reply README.md:28]: This issue is well resolved! ✔️

File: decoder.rb (hunk 1 above)
Comment (rloterh) [reply decoder.rb:64]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Zamir, Happen & Huseyin,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- Correct output decoded
- Good ReadMe

<br>
Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**